module/VuFindSearch/src/VuFindSearch/Service.php

```diff
@@ -375,6 +412,38 @@ function ($o) {
         return $this->backends[$backend];
     }
 
+    /**
+     * Perform a search and return a wrapped response.
+     *
+     * @param string              $backend Search backend identifier
+     * @param Query\AbstractQuery $query   Search query
+     * @param int                 $offset  Search offset
+     * @param int                 $limit   Search limit
+     * @param ParamBag            $params  Search backend parameters
+     * @param string              $context Event context
+     *
+     * @return RecordCollectionInterface
+     */
+    protected function performSearch($backend, Query\AbstractQuery $query,
```

Comment:
Is this performSearch method actually used for anything? I don't see any calls to it, but maybe I'm overlooking something.

Reply:
removed